.env.example

```diff
@@ -1,3 +1,3 @@
 EVENTS_API_URL=http://localhost:3000/api/mocked/crossweb
 SITE_URL=http://localhost:3000
-DISCORD_SERVER_ID=
\ No newline at end of file
+DISCORD_SERVER_ID=735954756169760768
```

Comment:
but it should be empty here?

Reply:
No, it's unnecessary empty. This data is public for everyone discord user, so we can provide it here. When user starting a project it's the easiest way to start